authz/README.md

```diff
@@ -95,7 +95,7 @@ python main.py
 
 ## Pre-requisites
 - 0MQ
-- OpenVINO&trade; 2021.3
+- OpenVINO&trade; 2020.4
```

Comment:
not require to mention about openvino

Reply:
fixed in bb9995e1a37b21ca5fbe7d13de2bfa0f470a692a